changes/changes-2.2.0.md

```diff
@@ -31,6 +31,7 @@ Apollo 2.2.0
 * [Fix the issue that namespace content being cleared when identical content is pasted into the namespace](https://github.com/apolloconfig/apollo/pull/4922)
 * [feat(openapi): allow user create app via openapi](https://github.com/apolloconfig/apollo/pull/4954)
 * [Support grayscale feature for non-properties namespaces](https://github.com/apolloconfig/apollo/pull/4952)
+* [Fix circular references on LdapAutoConfiguration](https://github.com/apolloconfig/apollo/pull/5055)
```

Comment:
The 2.2.0 was released. Please add the change to the https://github.com/apolloconfig/apollo/blob/master/CHANGES.md

Reply:
> The 2.2.0 was released. Please add the change to the https://github.com/apolloconfig/apollo/blob/master/CHANGES.md

added😃